### PR TITLE
Collapse DownloadControls into hoverable icon

### PR DESCRIPTION
**Pull request by adamabeshouse**

### CODE REVIEW

**Comment by alisman** [comment]
looks good to me.    my only wonder is if it would be better to implement as dropdown.  the way you've done this (and i suggested this), if this element is not absolutely positioned, it will bump other elements.   a dropdown (or flyout) avoids this problem because the dropdown itself is postioned absolutely and so can't affect position of other elements.

**Comment by adamabeshouse** [comment]
I dont understand what you're saying - the expansion is in a tooltip, which doesnt bump other elements

File: src/shared/components/downloadControls/DownloadControls.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.menu-item {
```
Comment (alisman) [inline src/shared/components/downloadControls/DownloadControls.css]: these need to be wrapped in .download-controls b/c they are global now

File: src/shared/components/downloadControls/DownloadControls.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    padding:5px;
+    user-select:none;
+}
+
+.menu-item.menu-item-enabled:hover {
+    cursor:pointer;
+    text-decoration:underline;
+}
+
+.menu-item.menu-item-disabled {
```
Comment (alisman) [inline src/shared/components/downloadControls/DownloadControls.css]: what's disabled for?

Comment (adamabeshouse) [reply src/shared/components/downloadControls/DownloadControls.css]: disabled is one of the options you can set

**Comment by adamabeshouse** [comment]
@alisman which tabs should we collapse for?

**Pull request merged**